adding managed properties (#527)
apache/incubator-kie-kogito-docs: serverlessworkflow/modules/ROOT/pages/cloud/operator/configuring-workflows.adoc

== Editing the Workflow Configuration

When the operator deploys the Workflow service, it also creates two `ConfigMap`s to store the runtime properties:

* The user properties are defined in a `ConfigMap` named after the `SonataFlow` object with the suffix `-props`. For example, if your Workflow name is `greeting`, then the `ConfigMap` name is `greeting-props`.

> * The xref:#managed-props[Managed Properties] are defined in a `ConfigMap` named after the `SonataFlow` object with the suffix `-managed-props`. For example, if your Workflow name is `greeting`, then the `ConfigMap` name is `greeting-props`.

Reviewer: ```suggestion
* The xref:#managed-props[Managed Properties] are defined in a `ConfigMap` named after the `SonataFlow` object with the suffix `-managed-props`. For example, if your Workflow name is `greeting`, then the `ConfigMap` name is `greeting-props-managed`.

? I don't remember the name exactly but those names differ, right?
```

Reviewer 2: `service-props` and `service-managed-props` is what I see on my deployed example

Reviewer 3: The generated name is  service-managed-props

Author: Updating it, thanks

> Note that managed properties always override any user property using the same key name and cannot be edited by the user: any change would be 
> overwritten by the operator at the next reconciliation cycle.

Reviewer: ```suggestion
[IMPORTANT]
====
Managed properties always override any user property using the same key name and cannot be edited by the user. Any change would be overwritten by the operator at the next reconciliation cycle.
====
```

Author: done thanks

|===

Other managed properties include:

> * The xref:cloud/quarkus/kubernetes-service-discovery.adoc[Kubernetes service discovery] properties

Reviewer: ```suggestion
* The xref:use-cases/advanced-developer-use-cases/service-discovery/kubernetes-service-discovery.adoc[Kubernetes service discovery] properties
```

> * The xref:data-index/data-index-quarkus-extension.adoc[Data Index location properties]

Reviewer: ```suggestion
* The xref:use-cases/advanced-developer-use-cases/data-index/data-index-quarkus-extension.adoc[Data Index location properties]
```

> * The xref:job-services/quarkus-extensions.adoc[Job Service location properties]

Reviewer: ```suggestion
* The xref:use-cases/advanced-developer-use-cases/job-service/quarkus-extensions.adoc[Job Service location properties]
```
